hackathon_site/dashboard/frontend/src/components/dashboard/TeamCard/TeamCard.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import TeamCard from "components/dashboard/TeamCard/TeamCard";
+import { render } from "@testing-library/react";
+import "@testing-library/jest-dom/extend-expect";
+import { fireEvent } from "@testing-library/react";
+import Cart from "../../../pages/Cart/Cart";
+import Teams from "../../../pages/Teams/Teams";
+
+describe("TeamCard", () => {
+    const members = ["Aki", "Graham", "Alex", "Raymond"];
+    const teamCode = "PAS3NLQ3";
+    test("place holder test", () => {
+        expect(teamCode).not.toBeNull();
+    });
+
+    // test("render", () => {
```

Comment:
uncomment & update these tests

Reply:
For the handle edit team test, can I remove that one since we are not using the edit button to change the teams

Reply:
keep it for now because we'll use it later, Leo's working on a ticket for the edit modal currently.

Reply:
can I remove the snapshot test, cause we got rid of the snapshots? @YuYing-Liang